README.md

```diff
@@ -26,6 +26,33 @@ make install
 ```
 
 
+
+## User data in the folder `.dsp-tools`
+
+DSP-TOOLS saves user data in the user's home directory, in the folder `.dsp-tools`. Here is an overview of its 
+structure:
+
+| folder     | command using it | description                                  |
+|:-----------|:-----------------|:---------------------------------------------|
+| xmluploads | `xmlupload`      | saves id2iri mappings and error reports      |
+| docker     | `stack-up`       | files necessary to startup Docker containers |
+
+
+
+## The `start-stack` command
+
+This command starts Docker containers of DSP-API and DSP-APP, in the version that is running on [https://admin.dasch.
+swiss/help](https://admin.dasch.swiss/help). In addition to the containers, a number of files from the DSP-API GitHub 
+repository is necessary. The version of the containers and these files must be the same. The version is hardcoded at the 
+following places in the code:
+
+- knora/dsplib/docker/docker-compose.yml: The 4 variables `services/{app,db,sipi,api}/image` must point to the 
+  DockerHub image of the last deployed version
+- knora/dsplib/utils/stack_handling.py: The variable `commit_of_used_api_version` must be the commit hash of DSP-API 
+  of the release that is running on [https://admin.dasch.swiss/help](https://admin.dasch.swiss/help).
```

Comment:
```suggestion
  of the version that is running on [https://admin.dasch.swiss](https://admin.dasch.swiss/help).
```

Reply:
There are a few inconsistencies in the vocabulary that you are using. IMO, it should be either a "release" that is  "deployed" on a server or a "version" that is "running" on a server. But maybe this is just me.